seeds/dex/trades/dex_trades_seed.csv

```diff
@@ -330,6 +330,8 @@ bnb,pancakeswap,3,2023-04-02 0:00,0x570f6d65a9f3046b184feb4efea90de552b31e6b06c1
 ethereum,kyberswap,elastic,2023-03-18,0x294c98fc947bf25331849d24ecc6dda039c293aa36b5046c88a8e677b1621d78,94,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,25060.144,0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0,12.515483515
 ethereum,kyberswap,elastic,2023-03-15,0x672f52ebebab645191403cfecb43dbb512100552e669288f4fc81174f9bdf587,104,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,15081.44,0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0,8.03308903
 ethereum,kyberswap,elastic,2023-01-09,0xa508a934395ab577132be907912b0d70c1493730696db951fc39e9b4506b9a96,37,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,5.222238778,0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0,4.767894198
+bnb,onepunchswap,quick,2023-03-03,0xa74ad784d8b0f673cb7ebe659f46d136a6d4a58bbd6386f3c4cc8a3adb064bdd,273,0x55d398326f99059ff775485246999027b3197955,1099.16920747,0x55d398326f99059ff775485246999027b3197955,60
```

Comment:
Hi @sunopar, please fix the unmatched seed issue:
https://github.com/duneanalytics/spellbook/actions/runs/4695694783/jobs/8325094069#step:10:273
https://dune.com/queries/2366396?d=1

test_description | result_model | expected_seed | block_date | blockchain | project | version | tx_hash | evt_index
-- | -- | -- | -- | -- | -- | -- | -- | --
equality test: token_sold_address | 0xe9e7cea3dedca5984780bafc599bd69add087d56 | 0x55d398326f99059ff775485246999027b3197955 | 2023-03-03 | bnb | onepunchswap | quick | 0xa74ad784d8b0f673cb7ebe659f46d136a6d4a58bbd6386f3c4cc8a3adb064bdd | 273